kaggle/ds.kaggle/deeplearning_questions.csv

```diff
@@ -0,0 +1,112 @@
+﻿ID,DESCRIPTION

```

Comment:
Is it a real data from kaggle? I so, could you please add the whole folder ds.kaggle to .gitignore. Describe what data is expected there in the read me. And keep only the code on how to process the data without committing intput and output files.
We're not supposed to submit data, so it would be great to keep our repository clean.